doc/user_guide/utilities.rst

```diff
@@ -96,3 +96,33 @@ wavelength and pixel position in the detector plane) follows the conventions
 described in the tutorial from  `Horiba Scientific
 <https://horiba.com/uk/scientific/products/optics-tutorial/wavelength-pixel-position>`_.
 
+Cropping multiple signals in the navigation axis 
+================================================
+
+The function :py:meth:`~.utils.axes.crop_edges`
+removes the specified number of pixels or % from the four edges of a spectral map,
+from the edges inwards. It takes a list of `Signals` and cropping can happen
+uniformly on all sides or by specifying the cropping range for each axis or each
+side. If the navigation axes shape across the list of signals is different, all
+signals can be rebinned to match the shape of the first signal in the list.
+It is a convenience wrapper for the ``inav`` `method in HyperSpy
+<https://hyperspy.org/hyperspy-doc/current/user_guide/signal.html#indexing>`_.
+
```

Comment:
In line with not deprecating the object oriented usage, add a sentence that the function can also be called directly as `s.crop_edges`, but then works only for a single signal.